plugins/collaajUpdatePlugin/lib/api/KalturaCaptureSpaceUpdateResponseInfoHash.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+class KalturaCaptureSpaceUpdateResponseInfoHash extends KalturaObject {
+    /**
+     * @var string
+     */
+    public $algorithm;

```

Comment:
if the value of that enum is eventually a sting such as "md5" that the client can "read" and use to run the relevant algorithm to verify the file authenticity - sure, no problem from integration perspective.